### ISSUE TITLE
Error message prints dimensions in wrong order? (migrated from Trac #1762)

### ISSUE THREAD

**Issue by thouis** [thouis/numpy-trac-migration#3312]
Original ticket http://projects.scipy.org/numpy/ticket/1762
Reported 2011-03-10 by atmention:charris, assigned to unknown.

```
ERROR: test_setitem (test_panel.TestWidePanel)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/usr/lib64/python2.7/site-packages/pandas/core/tests/test_panel.py", line 295, in test_setitem
    self.panel['ItemE'] = df
  File "/usr/lib64/python2.7/site-packages/pandas/core/panel.py", line 330, in __setitem__
    self.values[loc] = mat
ValueError: non-broadcastable output operand with shape (30,4) doesn't match the broadcast shape (4,30,1)
```

It appears to me that (4,30,1) should be (1,30,4)

**Comment by thouis** [comment]
Comment in Trac by atmention:mwiebe, 2011-03-10

Fixed in 9e28fce21a7bec4c9f1e.

**Comment by thouis** [comment]
Comment in Trac by atmention:mwiebe, 2011-05-31
